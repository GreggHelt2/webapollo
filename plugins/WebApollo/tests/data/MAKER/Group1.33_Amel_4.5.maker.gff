Group1.33	maker	gene	245454	247006	.	+	.	ID=maker-Group1%2E33-pred_gff_GNOMON-gene-4.137;Name=maker-Group1%252E33-pred_gff_GNOMON-gene-4.137;
Group1.33	maker	mRNA	245454	247006	.	+	.	ID=1:gnomon_566853_mRNA;Parent=maker-Group1%2E33-pred_gff_GNOMON-gene-4.137;Name=gnomon_566853_mRNA;_AED=0.45;_eAED=0.45;_QI=138|1|1|1|1|1|4|191|259;
Group1.33	maker	exon	245454	245533	.	+	.	ID=1:gnomon_566853_mRNA:exon:5976;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	exon	245702	245879	.	+	.	ID=1:gnomon_566853_mRNA:exon:5977;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	exon	246046	246278	.	+	.	ID=1:gnomon_566853_mRNA:exon:5978;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	exon	246389	247006	.	+	.	ID=1:gnomon_566853_mRNA:exon:5979;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	five_prime_UTR	245454	245533	.	+	.	ID=1:gnomon_566853_mRNA:five_prime_utr;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	five_prime_UTR	245702	245759	.	+	.	ID=1:gnomon_566853_mRNA:five_prime_utr;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	CDS	245760	245879	.	+	0	ID=1:gnomon_566853_mRNA:cds;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	CDS	246046	246278	.	+	0	ID=1:gnomon_566853_mRNA:cds;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	CDS	246389	246815	.	+	1	ID=1:gnomon_566853_mRNA:cds;Parent=1:gnomon_566853_mRNA;
Group1.33	maker	three_prime_UTR	246816	247006	.	+	.	ID=1:gnomon_566853_mRNA:three_prime_utr;Parent=1:gnomon_566853_mRNA;
