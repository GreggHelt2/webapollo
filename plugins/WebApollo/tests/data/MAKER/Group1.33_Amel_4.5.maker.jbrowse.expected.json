{
  "histograms" : {
    "stats" : [ {
      "basesPerBin" : "1000000",
      "max" : 1,
      "mean" : 1
    } ],
    "meta" : [ {
      "basesPerBin" : "1000000",
      "arrayParams" : {
        "length" : 1,
        "chunkSize" : 10000,
        "urlTemplate" : "hist-1000000-{Chunk}.json"
      }
    } ]
  },
  "featureCount" : 1,
  "intervals" : {
    "nclist" : [
    	     [0,245453,247006,1,"maker",null,"mRNA",null,"1:gnomon_566853_mRNA","gnomon_566853_mRNA",[[1,245453,245533,1,"maker",null,"exon",null,"1:gnomon_566853_mRNA:exon:5976",null,[]],[1,245701,245879,1,"maker",null,"exon",null,"1:gnomon_566853_mRNA:exon:5977",null,[]],[1,246045,246278,1,"maker",null,"exon",null,"1:gnomon_566853_mRNA:exon:5978",null,[]],[1,246388,247006,1,"maker",null,"exon",null,"1:gnomon_566853_mRNA:exon:5979",null,[]],[1,245453,245533,1,"maker",null,"five_prime_UTR",null,"1:gnomon_566853_mRNA:five_prime_utr",null,[]],[1,245701,245759,1,"maker",null,"five_prime_UTR",null,"1:gnomon_566853_mRNA:five_prime_utr",null,[]],[1,245759,245879,1,"maker","0","CDS",null,"1:gnomon_566853_mRNA:cds",null,[]],[1,246045,246278,1,"maker","0","CDS",null,"1:gnomon_566853_mRNA:cds",null,[]],[1,246388,246815,1,"maker","1","CDS",null,"1:gnomon_566853_mRNA:cds",null,[]],[1,246815,247006,1,"maker",null,"three_prime_UTR",null,"1:gnomon_566853_mRNA:three_prime_utr",null,[]]]],
    ],
    "classes" : [ {
      "isArrayAttr" : {
        "Subfeatures" : 1
      },
      "attributes" : [ "Start", "End", "Strand", "Source", "Phase", "Type", "Score", "Id", "Name", "Subfeatures" ]
    }, {
      "isArrayAttr" : {
      },
      "attributes" : [ "Start", "End", "Strand", "Source", "Phase", "Type", "Score", "Id", "Name", "Subfeatures" ]
    }, {
      "isArrayAttr" : {
        "Sublist" : 1
      },
      "attributes" : [ "Start", "End", "Chunk" ]
    } ],
    "maxEnd" : 247006,
    "count" : 1,
    "lazyClass" : 2,
    "urlTemplate" : "lf-{Chunk}.json",
    "minStart" : 245453
  },
  "formatVersion" : 1
}
